README.md

```diff
@@ -1,22 +1,45 @@
-﻿# Project Title
+﻿# Chinook.DynamicMvvm
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-3-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
-{Project tag line}
+The `Chinook.DynamicMvvm` library assists in MVVM (Model - View - ViewModel) development.
 
-{Small description of the purpose of the project}
+There are 3 main components to `Chinook.DynamicMvvm`:
+
+* [IViewModel](src/DynamicMvvm.Abstractions/ViewModel/IViewModel.md): a store of `IDisposable` that can be bound to a view and is meant to contain a set of `IDynamicProperties` and `IDynamicCommands`.
+* [IDynamicProperty](src/DynamicMvvm.Abstractions/Property/IDynamicProperty.md): a property that will notify its subscribers when its value changes.
+* [IDynamicCommand](src/DynamicMvvm.Abstractions/Command/IDynamicCommand.md): a `ICommand` that will notify its subscribers when it is executing.
 
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
 
 ## Getting Started
 
-{Instructions to quickly get started using the project: pre-requisites, packages
-to install, sample code, etc.}
+Add the `Chinook.DynamicMvvm` nuget package and its dependencies to your project.
+
+If your project uses `Reactive` concepts, we recommend installing `Chinook.DynamicMvvm.Reactive` as well, to gain access to `IDynamicProperty` and `IDynamicCommand` extension methods.
```

Comment:
Instead of "Reactive concepts", I think we can mention Reactive extensions (as a link) directly.